pika/channel.py

```diff
@@ -457,6 +457,8 @@ def basic_qos(self,
         :param bool all_channels: Should the QoS apply to all channels
```

Comment:
Unrelated to this PR, but might be good to clarify in "The prefetch-count is ignored if the no-ack option is set." above that this applies to consumers. Perhaps word it as follows: "The prefetch-count is ignored by consumers with the no-ack option turned on."